utils/farms.ts

```diff
@@ -10,14 +32,5 @@ export const fetchTokenPrice: (
     if (!provider || !poolInfo_) {
         return [new BigNumber(1)];
     }
-
-    const poolInfo = await Pool.Create({
-        ...poolInfo_,
-        provider,
-    });
-
-    return tokenAddresses.map((tokenAddress) => {
-        const isLong: boolean = tokenAddress.toLowerCase() === poolInfo.longToken.address.toLowerCase();
-        return isLong ? poolInfo.getNextLongTokenPrice() : poolInfo.getNextShortTokenPrice();
-    });
+    return tokenAddresses.map((_tokenAddress) => new BigNumber(1));
```

Comment:
maybe add a TODO here or something so we don't forget

Reply:
aight